### PR TITLE
[plugins/user/procbyuser] Added plugin to monitor processes of a user

### PR DESCRIPTION
**Pull request by PoroCYon**
Graphs the number of processes owned by a user over time

### CODE REVIEW

File: plugins/user/procbyuser [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+#	config		(required)
+#	autoconf	(optional - used by munin-config)
+#
+
+#%# family=auto
+#%# capabilities=autoconf
+
+. "$MUNIN_LIBDIR/plugins/plugin.sh"
+
+#[ -z "$OTHER_FIELD" ] && OTHER_FIELD="others"
```
Comment (sumpfralle) [inline plugins/user/procbyuser:25]: Please remove commented code.

File: plugins/user/procbyuser [hunk 2]
```diff
@@ -0,0 +1,80 @@
+#!/bin/sh
+#
+# Plugin to monitor the amount of processes owned by a user
```
Comment (sumpfralle) [inline plugins/user/procbyuser:3]: Please use a perldoc header (look for `=cut` in other shell plugins for examples).

File: plugins/user/procbyuser (hunk 1 above)
Comment (PoroCYon) [reply plugins/user/procbyuser]: Done.

File: plugins/user/procbyuser (hunk 2 above)
Comment (PoroCYon) [reply plugins/user/procbyuser]: Done.

**Comment by sumpfralle** [comment]
Thanks!

**Pull request merged**